### PR TITLE
Replace non-commercial icons

### PR DESCRIPTION
**Pull request by guoyunhe**
This change is [Reviewable](https://reviewable.io/reviews/citra-emu/citra/5093)

### CODE REVIEW

**Comment by zhaowenlan1779** [comment]
Aren't these icons used while verifying the token? For instance https://github.com/citra-emu/citra/blob/f106e76132db19344ae5353fcd4a143700b5158d/src/citra_qt/configuration/configure_web.cpp#L114-L124

**Comment by guoyunhe** [comment]
Oh, my mistake. Then I replaced them with icons8 icons (they say they open source icons for open source projects).

**Comment by zhaowenlan1779** [comment]
To be fairly honest, I don't see why we have to replace them. If it is for licensing, then while non-commerical conflicts with GPLv2, CC-BY-ND conflicts with it as well, and that's why we have separate license notices for icons. Additionally, when I retrieved the other icons, I was pretty sure there was a statement saying the icons were licensed under CC-BY-ND-3.0, but now I can't find that anymore, only a 'free' proprietary license.

**Comment by guoyunhe** [comment]
To submit the package to Linux distribution repos, the license must be compatible. CC-BY-ND is okay but non-commercial is not. With non-commercial license, you cannot submit the software into any app store. But with CC-BY-ND, you can. It brings a lot more freedom.

Here are also other options. Like https://remixicon.com/ is licensed under MIT license.

File: dist/license.md [hunk 1]
```diff
@@ -15,11 +15,9 @@ qt_themes/default/icons/48x48/folder.png | CC BY-ND 3.0 | https://icons8.com
 qt_themes/default/icons/48x48/no_avatar.png | CC BY-ND 3.0 | https://icons8.com
 qt_themes/default/icons/48x48/plus.png | CC0 1.0 | Designed by BreadFish64 from the Citra team
 qt_themes/default/icons/48x48/sd_card.png | CC BY-ND 3.0 | https://icons8.com
-qt_themes/qdarkstyle/icons/16x16/checked.png | Free for non-commercial use
```
Comment (jroweboy) [inline dist/license.md]: Did we have different icons for default and dark themes before?

**Comment by KAMiKAZOW** [comment]
"ND" means that derivative works are not allowed, therefore they are non-free and therefore forbidden in the default repositories of all major Linux distributions.

**Pull request merged**